Pillow sample on macOS

**To Reproduce**
Build and run the pillow sample on macOS

**Expected behavior**
Build and run.

**Desktop (please complete the following information):**
 - Platform information (e.g. Ubuntu Linux 16.04): macOS
 - OS architecture (e.g. amd64): 64
 - cx_Freeze version [e.g. 6.6]: 6.6 / 6.7dev
 - Python version [e.g. 3.9]: 3.6-3.9

**Additional context**
Part of my log:
```
2021-04-24T15:53:38.8672750Z *** WARNING ***
2021-04-24T15:53:38.8673820Z Attempting to copy two files to 'build/exe.macosx-10.14-x86_64-3.8/liblzma.5.dylib'
2021-04-24T15:53:38.8675830Z source 1: '/Users/runner/hostedtoolcache/Python/3.8.9/x64/lib/python3.8/site-packages/PIL/.dylibs/liblzma.5.dylib' (real: '/Users/runner/hostedtoolcache/Python/3.8.9/x64/lib/python3.8/site-packages/PIL/.dylibs/liblzma.5.dylib')
2021-04-24T15:53:38.8677720Z source 2: '/usr/local/opt/xz/lib/liblzma.5.dylib' (real: '/usr/local/Cellar/xz/5.2.5/lib/liblzma.5.dylib')
2021-04-24T15:53:38.8678680Z (This may be caused by including modules in the zip file that rely on binary libraries with the same name.)
2021-04-24T15:53:38.8679370Z Using only source 1.
2021-04-24T15:53:38.9058480Z + popd
2021-04-24T15:53:38.9059980Z + echo 'Run pillow sample'
2021-04-24T15:53:38.9060930Z ~/work/freeze-test/freeze-test
2021-04-24T15:53:38.9061470Z + TEST_PRECMD=./
2021-04-24T15:53:38.9062290Z + '[' darwin19 == msys ']'
2021-04-24T15:53:38.9063490Z + BUILD_DIR=/Users/runner/work/freeze-test/freeze-test/cx_Freeze/samples/pillow/build/exe.macosx-10.14-x86_64-3.8
2021-04-24T15:53:38.9065380Z + pushd /Users/runner/work/freeze-test/freeze-test/cx_Freeze/samples/pillow/build/exe.macosx-10.14-x86_64-3.8
2021-04-24T15:53:38.9066180Z + count=0
2021-04-24T15:53:38.9066550Z Run pillow sample
2021-04-24T15:53:38.9067990Z ~/work/freeze-test/freeze-test/cx_Freeze/samples/pillow/build/exe.macosx-10.14-x86_64-3.8 ~/work/freeze-test/freeze-test
2021-04-24T15:53:38.9071230Z ++ python /Users/runner/work/freeze-test/freeze-test/ci/build-test-json.py pillow 0
2021-04-24T15:53:38.9496640Z + TEST_NAME=test_pillow
2021-04-24T15:53:38.9498500Z + '[' -z test_pillow ']'
2021-04-24T15:53:38.9499260Z + [[ test_pillow == gui:* ]]
2021-04-24T15:53:38.9499760Z + [[ test_pillow == svc:* ]]
2021-04-24T15:53:38.9500160Z + ./test_pillow
2021-04-24T15:53:39.6799760Z Traceback (most recent call last):
2021-04-24T15:53:39.6803420Z   File "/Users/runner/hostedtoolcache/Python/3.8.9/x64/lib/python3.8/site-packages/cx_Freeze/initscripts/__startup__.py", line 81, in run
2021-04-24T15:53:39.6805290Z     module.run()
2021-04-24T15:53:39.6807130Z   File "/Users/runner/hostedtoolcache/Python/3.8.9/x64/lib/python3.8/site-packages/cx_Freeze/initscripts/Console.py", line 36, in run
2021-04-24T15:53:39.6808410Z     exec(code, m.__dict__)
2021-04-24T15:53:39.6809390Z   File "test_pillow.py", line 4, in <module>
2021-04-24T15:53:39.6810480Z   File "<frozen zipimport>", line 259, in load_module
2021-04-24T15:53:39.6812330Z   File "/Users/runner/hostedtoolcache/Python/3.8.9/x64/lib/python3.8/site-packages/PIL/Image.py", line 109, in <module>
2021-04-24T15:53:39.6813560Z     from . import _imaging as core
2021-04-24T15:53:39.6815920Z ImportError: dlopen(/Users/runner/work/freeze-test/freeze-test/cx_Freeze/samples/pillow/build/exe.macosx-10.14-x86_64-3.8/lib/PIL._imaging.cpython-38-darwin.so, 2): Library not loaded: @loader_path/.dylibs/libopenjp2.2.4.0.dylib
2021-04-24T15:53:39.6818750Z   Referenced from: /Users/runner/work/freeze-test/freeze-test/cx_Freeze/samples/pillow/build/exe.macosx-10.14-x86_64-3.8/lib/PIL._imaging.cpython-38-darwin.so
2021-04-24T15:53:39.6820280Z   Reason: image not found
2021-04-24T15:53:39.6873860Z ##[error]Process completed with exit code 255.
```
@cainesi You can test in your enviroment and maybe find a solution?

--- a/cx_Freeze/hooks/__init__.py
+++ b/cx_Freeze/hooks/__init__.py
@@ -317,11 +317,6 @@ def load_pikepdf(finder: ModuleFinder, module: Module) -> None:
     finder.include_package("pikepdf")
 
 
-def load_PIL(finder: ModuleFinder, module: Module) -> None:
-    """The Pillow must be loaded as a package."""
-    finder.include_package("PIL")
-
-
 def load_plotly(finder: ModuleFinder, module: Module) -> None:
     """The plotly must be loaded as a package."""
     finder.include_package("plotly")

--- a/cx_Freeze/hooks/pil.py
+++ b/cx_Freeze/hooks/pil.py
@@ -1,0 +1,20 @@
+"""A collection of functions which are triggered automatically by finder when
+Pillow (PIL) package is included.
+"""
+
+from __future__ import annotations
+
+from .._compat import IS_MACOS
+from ..finder import ModuleFinder
+from ..module import Module
+
+
+def load_pil(finder: ModuleFinder, module: Module) -> None:
+    """The Pillow must be loaded as a package."""
+    finder.include_package("PIL")
+
+    # [macos] copy dependent files when using zip_include_packages
+    if IS_MACOS:
+        source_dir = module.file.parent / ".dylibs"
+        if source_dir.exists() and module.in_file_system == 0:
+            finder.include_files(source_dir, "lib/.dylibs")

